app/policies/client_policy.rb

```diff
@@ -20,7 +20,9 @@ def resolve
   alias_method :need_accompanying?, :superadmin_or_department_manager?
 
   # suplementary policies
-  alias_method :supervisor?, :superadmin?
-  alias_method :journals?,   :superadmin?
-  alias_method :comments?,   :superadmin?
+  alias_method :supervisor?,       :superadmin?
+  alias_method :journals?,         :superadmin?
```

Comment:
Since all those methods refer to the superadmin, why can't we have just one method?

Reply:
to be able to control visibility explicitly in views, and also change those. Otherwise we end up asking `policy(Client).destroy?` for something in the views that has absoluteley nothing to do with destroy again.

Reply:
I didn't suggest using `policy(Client).destroy?` to control view visibility. I suggested having one method eg `supervisor?` for all view visibility checks, since all of them check for the superadmin. After a while we won't remember which type of user should have access to those fields, unless we look up the client policy.

Reply:
The approach would be that one would not worry about who can see what within a view. So within the view I would, if the Model-Field has any restrictions at all, ask policy if it should be displayed, and the granular tuning of who does what is done in the policy. If they are all the same now, it doesn't necessary stay like this forever. So if the view is built opon `policy(Client).state?` = Tell me if state should be displayed or not.

I don't state that the approach you suggest would be wrong. 
The approach I tried to implement was just to keep policy decisions as much as possible outside the views and within the policy. So It would be rather a decision if we pray in `app/views/clients/_form.html.slim` or in `app/policies/client_policy.rb`.
Dont argue about praying 😉

Reply:
Then unless something needs to change regarding [this](https://github.com/panter/aoz-003/pull/315#discussion_r148503456), then is it ready to be merged?

Reply:
mother tongue is beeing displayed in the colum before, and as you requested me to transfer the test for empty language levels in view displayed, it made this neeeded, because the scope used before was obviously not delivering `language without a level`. So Either you have your test or you have what ever you have!?!

Reply:
I am only referring to the addition of `.not_german` here:

`td = render 'language_skills/show_inline', language_skills: volunteer.language_skills.not_german `

So the question was why:
`td = render 'language_skills/show_inline', language_skills: volunteer.language_skills.not_german`

and why not:

`td = render 'language_skills/show_inline', language_skills: volunteer.language_skills`

I don't see how that has anything to do with the test.

Reply:
you know what. I'm dropping that test and put the index back to normal